### DISCUSSION TITLE
Access to Filesystem location

### DISCUSSION THREAD

**Discussion by kesch095** [photoprism/photoprism#1985] [Q&A] [answered]
Hi everyone,

I just setup Photoprism na dimported all my existing Photos. I used the import folder for doing that so PhotoPrism moved them to directories according to the date they were created.

Now there are some photos I'd like to move off of my server. Is there a possibility for Photoprism to show me the location on the filesystem for these photos? 
Or is there no other way other than to delete them using Photoprism?

Kind regards,
Kevin

**Comment by graciousgrey** [comment, marked as answer]
When using import PhotoPrism copies or moves your files to originals/year/month e.g. originals/2022/01.
You find the exact path + name on the files tab of the edit dialog (https://docs.photoprism.app/user-guide/organize/edit/#edit-files).

You can remove files directly from your originals folder using the filesystem.

**Comment by kesch095** [comment]
Thanks for your help! As there are many photos that i want to move this way I thought that there maybe is a better way than moving them one by one. 

I also figured out about the sharing feature and added my nextcloud instance as a webdav server. Are the exports the same as the originals regarding the metadata? I could use this way then.

**Reply by lastzero** [reply]
When uploading files via WebDAV, it depends on your settings. You can select "original" (with metadata) or a downscaled size (without metadata).

**Reply by kesch095** [reply]
I'll do it this way then, thank you! :)
